cmd/chloggen/main_test.go

```diff
@@ -279,6 +287,28 @@ func breakingEntry() *Entry {
 	}
 }
 
+func entryWithSubtext() *Entry {
+	lines := []string{
+		"  - foo",
+		"    - bar",
+		"  - blah",
+		"    - 1234567",
+	}
+	var subtext string
+	if runtime.GOOS == "windows" {
+		subtext = strings.Join(lines, "\r\n")
+	} else {
+		subtext = strings.Join(lines, "\n")
+	}
```

Comment:
ditto here, I don't quite get why we want a different output on Windows; with any editor newlines without `\r` should work fine, and we would avoid platform specific code

Reply:
The problem is ultimately that the golden files used in tests (`testdata/*.md`) actually have the `\r` on windows, so something like this is necessary to pass tests. Alternately, I can 1) skip these tests on Windows, or 2) probably can clean the expected `[]byte` that results from reading in the files on windows.